sigstore-java/src/main/java/dev/sigstore/tuf/model/SnapshotMeta.java

```diff
@@ -29,6 +31,8 @@
 @Value.Immutable
 public interface SnapshotMeta extends TufMeta {
 
+  Integer DEFAULT_MAX_LENGTH = 2000000;
```

Comment:
just curious where the default comes from?

Reply:
oh guess I never added the comment, it comes from the python-tuf defaults.